mlprimitives/jsons/dsp.stdDevFreqAnalysis.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "dsp.stdDevFreqAnalysis",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",
+    "documentation": "",
+    "description": "Anomalies detection in satellite telemetry data using the standard deviation of the frequency components calculated using time-dependent Fast Fourier Transform",
+    "modalities": [],
+    "primitive": "mlprimitives.dsp.freqAnalysis",
+    "fit": {
+        "method": "fitFreqStdDev",        
+        "args": [
+            {
+                "name": "X",
+                "description": "training signal",
```

Comment:
Please capitalize, "Training signal"  to make it more professional

Reply:
This JSON file is not used anymore, I'll remove it.